service/geopmdpy_test/TestActiveSessions.py

```diff
@@ -162,155 +162,6 @@ def create_json_file(self, directory, filename, contents, permissions=0o600):
             else:
                 json.dump(contents, file)
 
-    def test_default_creation(self):
```

Comment:
Please leave these tests.  They should continue to pass without modification, and we will update them later to mock the secure_*() methods